todolist.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            To-Do List
+        </title>
+        <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+    </head>
+    <body>
+        <div id='header'>
```

Comment:
these ids and classes currently don't do anything (in this PR there is no functionality or css related to these). would suggest removing